CoEpi/ui/home/HealthQuizViewController.swift

```diff
@@ -1,30 +1,56 @@
-//
-//  HealthQuizViewController.swift
-//  CoEpi
-//
-//  Created by Dusko Ojdanic on 3/26/20.
-//  Copyright © 2020 org.coepi. MIT licensed: see LICENSE file.
-//
-
 import UIKit
```

Comment:
@johnsonh There are 2 VCs with the same name: CoEpi/ui/home/HealthQuizViewController.swift and CoEpi/ui/healthquiz/HealthQuizViewController.swift

Reply:
```app-ios duskoo$ find . -name HealthQuizViewController.swift
./CoEpi/ui/healthquiz/HealthQuizViewController.swift
./CoEpi/ui/home/HealthQuizViewController.swift
```